### ISSUE TITLE
az network lb frontend-ip create will sometimes not honor the --private-ip-address parameter

### ISSUE THREAD

**Issue by hectoralicea** [Azure/azure-cli#6607]
**Describe the bug**
The command `az network lb frontend-ip create` sometimes it will not honor the `--private-ip-address` parameter but rather assigns it some random IP in the same subnet.

**To Reproduce**
Not sure if it's limited to my large config or if its reproducible in a simple config

**Expected behavior**
IPs assigned to the frontend IP configuration should be the IPs provided in the `--private-ip-address` parameter 

**Environment summary**
Install Method (pip) / CLI version (azure-cli (2.0.26)) / OS version (CentOS Linux release 7.4.1708 (Core) ) / Shell Type (bash)

**Additional context**

**Comment by hectoralicea** [comment]
Got it to work by adding a ```az network lb frontend-ip update```  command following the ```az network lb frontend-ip create``` .  So my code looks like this:

```
az network lb frontend-ip create --lb-name ${lb_name} --name ${frontend_ip_2} --resource-group ${RG} --private-ip-address ${webintranet_lb_ip} --subnet ${subnetid}
az network lb frontend-ip update --lb-name ${lb_name} --name ${frontend_ip_2} --resource-group ${RG} --private-ip-address ${webintranet_lb_ip} --set privateIpAllocationMethod=Static ; sleep 3

```
in cases where its the second frontend-ip being created I had to add the update command to ensure that the private ip sticks and that the privateIpAllocationMethod sets to Static.  Running debug before the update command is set, the status of the private-ip-address is that its assigned to the lowest IP available in the subnet, and the privateIpAllocationMethod=Dynamic.  After the update command, the IP changes to what I wanted it to be set to in the first command and the privateIpAllocationMethod=Static

**Comment by Dagon-** [comment]
Just ran into this issue.
Same symptoms and work around as provided by @hectoralicea 
Currently on azure-cli 2.0.45

**Comment by adewaleo** [comment]
@Dagon-, @hectoralicea, I couldn't reproduce this issue by creating a simple load balancer frontend. 

Could you give more information? For example, what your virtualnetwork creation command looks like.

**Comment by adewaleo** [comment]
This issue has been fixed in #7232 and should be reflected in the next release of the CLI.

I couldn't reproduce this bug because I was working off of the lates dev release which already addressed the bug.

I will be closing this issue, feel free to let me know if you have any more questions?
